recipes/hunspell/all/CMakeLists.txt

```diff
@@ -28,6 +26,8 @@ add_library(
     "${src}/suggestmgr.cxx"
 )
 
+target_compile_features(hunspell PRIVATE cxx_std_11)
```

Comment:
Is it required to only the latest version or any older too?

Reply:
C++11 is the oldest that is supported. If I remember correctly the library uses `auto` which is a C++11 feature.